incubator/hnc/pkg/kubectl/tree.go

```diff
@@ -34,19 +34,32 @@ var (
 var treeCmd = &cobra.Command{
 	Use:   "tree",
 	Short: "Displays the hierarchy tree rooted at the given namespace",
-	Args:  cobra.ExactArgs(1),
+	Args:  cobra.MaximumNArgs(1),
 	Run: func(cmd *cobra.Command, args []string) {
-		nnm := args[0]
+		var nnm string
+		if len(args) != 0 {
+			nnm = args[0]
+		}
```

Comment:
You don't really need this - below, where you would have said `if nnm != ""`, just say `if len(args) != 0` instead